InferenceSystem/src/globals.py

```diff
@@ -4,6 +4,8 @@
     "orcasound_lab": "https://s3-us-west-2.amazonaws.com/streaming-orcasound-net/rpi_orcasound_lab",
     "port_townsend": "https://s3-us-west-2.amazonaws.com/streaming-orcasound-net/rpi_port_townsend",
     "bush_point": "https://s3-us-west-2.amazonaws.com/streaming-orcasound-net/rpi_bush_point",
+    "sunset_bay": "https://s3-us-west-2.amazonaws.com/streaming-orcasound-net/rpi_sunset_bay",
```

Comment:
Likewise, can we move this into a config rather than something hardcoded?

Reply:
Will take this up as a future PR